main.js

```diff
@@ -1,4 +1,172 @@
+  }];
+
+/* Card generator */
+function displayProjects() {
+  projectSection.innerHTML = cardItemsData.map((projectCard) => `
+        <!--card 1-->
+        <article id=project-card${projectCard.id} class="projects__card">
+            <div class="projects__card--img">
+                <img src="${projectCard.image}" alt="project image">
+            </div>
+            <div class="projects__card--text">
+                <h3 class="projects__card--title">${projectCard.title}</h3>
+                <div class="projects__card--tags">
+                    <ul>
+                        <li>${projectCard.technologies[0]}</li>
+                        <li>${projectCard.technologies[1]}</li>
+                        <li>${projectCard.technologies[2]}</li>
+                        <li>${projectCard.technologies[3]}</li>
+                    </ul>
+                </div>
+                <button data-modal-target="#modal${projectCard.id}" class="action-btn">See project</button>
+            </div>
+        </article>
+    <!--Modal window-->
+    <article class="projects__modal--conatiner" id="modal${projectCard.id}">
+        <div class="projects__modal--body">
+            <button data-close-modal class="close-modal-desk">&times;</button>
+            <div class="projects__modal--hero">
+                <button data-close-modal class="close-modal">&times;</button>
+            </div>
+            <h1 class="projects__modal--title">${projectCard.title}</h1>
+            <div class="projects__modal--tags">
+                <ul>
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                    <li class="projects__modal--tags-boot">Boostrap</li>
+                    <li class="projects__modal--tags-ter">Terminal</li>
+                    <li class="projects__modal--tags-code">Codepen</li>
+                </ul>
+            </div>
+            <div class="projects__modal--text">
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+                    been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                    galley of type and scrambled it 1960s. <br>
+                    <br> Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum
+                    has
+                    been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                    galley of type and scrambled it 1960s.
+                </p></span>
+            </div>
+            <div class="projects__modal--actions">
+                <button class="action-btn">See live<img src="./assets/modal_see live icon.svg" alt=""></button>
+                <button class="action-btn">See source<img src="./assets/modal_Icon -GitHub.svg" alt=""></button>
+            </div>
+        </div>
+    </article>
+        `).join('');
+}
```

Comment:
- [ ] You did well on this project milestone but the modal popup on mobile view has some issues. 
    - The close icon is not visible 
    - The See Project and See source Buttons are hidden below, as such you should add a scroll functionality so users can scroll down to the buttons. 
    - The image size is not the same as observed in the Figma design.
    - The background of the modal should have some blur effect so as to improve focus on the modal content.
    - The padding on all sides of the document should be a little bit higher than what you have now. Please check with Figma template and get the correct padding on all sides.

your work | Figma design
---- | ----
<img width="293" alt="image" src="https://github.com/cuauh-cabrera/my_portfolio/assets/84156714/99ee277a-4a3f-48b5-ba89-9f41e8587d5a"> | ![image](https://github.com/cuauh-cabrera/my_portfolio/assets/84156714/b643d219-627b-4ec3-b1ee-9869dbe44a0a)


Your modal on mobile view should look exact like the Figma design as much as possible. Kindly fix the issues.